build-system/test-configs/config.js

```diff
@@ -130,7 +130,7 @@ const lintGlobs = [
  *  - forbidden terms: local/no-forbidden-terms
  */
 const presubmitGlobs = [
-  '**/*.{css,go}',
+  '**/*.{css,go,md}',
```

Comment:
I just realized we're not running `lint` and `presubmit` during CI when the rules or implementation change. We're only running when one of the files that needs to be checked is modified. I've done this in #33714. Can we rebase this PR on that to make sure the checks pass before submitting?

Reply:
I missed your comment before I merged, apologies! Though I've confirmed that both changes in `main` maintain `amp presubmit` passing.

Reply:
Perfect! (And no worries, I was just trying to be extra careful.)